src/demo/pages/Home.tsx

```diff
@@ -1,50 +1,18 @@
-import React, { useEffect } from "react";
-
-import { api, SdkError, SdkErrors, ExternalUser } from "@tonomy/tonomy-id-sdk";
+import React from "react";
+import { api } from "@tonomy/tonomy-id-sdk";
 import settings from "../../common/settings";
 import "./Home.css";
-import { TH1, TH3, TP } from "../../common/atoms/THeadings";
+import { TP } from "../../common/atoms/THeadings";
 import logo from "/tonomy-logo48.png";
 import Rectangle from "../assets/Rectangle.png";
 import HandImage from "../assets/handImg.png";
-import { Highlighter } from "rc-highlight";
 import "@tonomy/tonomy-id-sdk/build/api/tonomy.css";
-import { useNavigate } from "react-router-dom";
-import useErrorStore from "../../common/stores/errorStore";
 
 export default function Home() {
-  const errorStore = useErrorStore();
-
   async function onButtonPress() {
     api.ExternalUser.loginWithTonomy({ callbackPath: "/callback" });
   }
 
-  const navigation = useNavigate();
-
-  async function onRender() {
-    try {
```

Comment:
I think we still need to check for user here (from the AuthProvider) and if found redirect them to the /user-home page.
similar for the /callback page